src/firebase/index.js

```diff
@@ -0,0 +1,27 @@
+import { initializeApp } from "firebase/app";
+import { getAuth } from "firebase/auth";
+import { collection, getFirestore } from "firebase/firestore";
+import { getStorage } from "firebase/storage";
+
+const firebaseConfig = {
+  apiKey: process.env.VUE_APP_API_KEY,
+  authDomain: process.env.VUE_APP_AUTH_DOMAIN,
+  projectId: process.env.VUE_APP_PROJECT_ID,
+  storageBucket: process.env.VUE_APP_STORAGE_BUCKET,
+  messagingSenderId: process.env.VUE_APP_MESSAGING_SENDER_ID,
+  appId: process.env.VUE_APP_ID,
+  measurementId: process.env.VUE_APP_MEASURMENT_ID,
+};
+
+
+const firebaseapp = initializeApp(firebaseConfig);
+
+const db = getFirestore(firebaseapp);
+
+const todoRef = collection(db, "testCollection");
```

Comment:
do you need this todoRef? @aayushbatra02

Reply:
sorry sir, I was using this for testing purposes